integration-tests/lra/src/test/java/org/apache/camel/quarkus/component/lra/it/LraTestResource.java

```diff
@@ -36,8 +42,16 @@
     @Override
     public Map<String, String> start() {
         try {
+            String hostname = InetAddress.getLocalHost().getHostName();
```

Comment:
Small nit - I don't see `hostname` used for anything so I guess it can be removed?

Reply:
Hi @jamesnetherton ,

Thanks for all the feedback!
I sent a new commit which removed unused code you mentioned above.

Freeman